docs/tutorials.rst

```diff
@@ -42,7 +42,7 @@ CAP-client
 -------------
 
 The CAP-client is a command-line tool for preserving analyses.
-It is implemented as a python package and its documentation can be found `here <https://cap-client.readthedocs.io>`_.
+It is implemented as a python package and its documentation can be found `here <https://cap-client.readthedocs.io>`__.
```

Comment:
Should be only one trailing `_` in reST links....

Reply:
hmmmm, not sure about that, if I have it only one is failing with this error 
https://github.com/sphinx-doc/sphinx/issues/3921

Reply:
@tiborsimko ok you are right, was failing when I tried to build it locally but seems that it is not an issue. I updated the PR.

Reply:
Ah, if the link target is the same, you can define it once and use it multiple times like:

```

This is `foo`_ and...

As we said there is also `foo`_ that you can take advantage of ...

Yet another paragraph where we mention `foo`_ because...

.. _foo: http://example.org
```

Reply:
Ahh cool, didnt know that, will update !

Reply:
But actually the links are different so it shouldnt be an issue.

Reply:
In this case we shouldn't really call them both "here":

```console
$ rm -rf docs/_build/
$ python setup.py build_sphinx
[...]
docs/tutorials.rst:2: WARNING: Duplicate explicit target name: "here".
```

Better give them different names...

Reply:
http://uxmovement.com/content/why-your-links-should-never-say-click-here/

Reply:
Ok cool! Updated!